raven/event.go

```diff
@@ -40,6 +43,14 @@ func NewHttp(req *http.Request) *Http {
 	}
 }
 
+func DsnData(dsn string) interface{} {
```

Comment:
just thinking about how this is going to be used

```go
glog.Error("something bad happened", raven.DsnData(consulting))
```

That's pretty good, maybe one tweak?

```go
glog.Error("something bad happened", raven.AltDsn(consulting))
```

or

```go
glog.Error("something bad happened", raven.Dsn(consulting))
```


I don't feel strongly though, up to you